public/index.html

```diff
@@ -24,7 +24,7 @@
 <body class="h-100" onresize="fit()" onload="fit();updateGutterVisibility();">
     <div id="preloader"><img src="images/preloader.gif" width="100px"></div>
 
-    <div id="login" style="display:inline">
+    <div id="login" style="display:none">
```

Comment:
Why is this file needed at all? Shouldn't it just use the file imported from the education platform repository?

Reply:
For the dockerised playground there there is some redirect functionality to the list of example activities if no activity file is specified that is added to the html. This could be done differently to avoid the duplication of the playground index.html.